### DISCUSSION TITLE
Version to use when packaging

### DISCUSSION THREAD

**Discussion by ghost** [aditya-K2/gomp#50] [Q&A] [answered]
Currently, there does not seem to be any releases, but I wish to package this software. So should I wait for a real release or just use the date of commit like in the aur?

**Comment by akurdunkar** [comment, marked as answer]
Yes.

**Reply by ghost** [reply]
I am confused, which one of my suggestions are you agreeing with?

**Reply by akurdunkar** [reply]
I am sorry for the confusion. Please use the version that aur follows. There are some features that I want to implement before releasing semantic version 1.0.0. Until then please use the aur version.


BTW what are you packaging gomp for?

**Reply by ghost** [reply]
> BTW what are you packaging gomp for?

Alpine linux

I also need to know if the project is licensed GPL-3.0-or-later or GPL-3.0-only.

> There are some features that I want to implement before releasing semantic version 1.0.0.

That's fine, but preferably there would be a pre-1.0 release such as 0.1, 0.2, 0.2.1, etc.

**Reply by akurdunkar** [reply]
>That's fine, but preferably there would be a pre-1.0 release such as 0.1, 0.2, 0.2.1, etc.

I see. If I get time this weekend. I will try to release the pre-1.0 release. Until then please use the aur version. 

> I also need to know if the project is licensed GPL-3.0-or-later or GPL-3.0-only.

GPL-3.0

**Reply by ghost** [reply]
> GPL-3.0

The license should be one from the [SPDX License List](https://spdx.org/licenses/) (This is the case for many package repositories). `GPL-3.0-only` is probably the one you are referring to, as it doesn't allow people to release the software under future versions of the gpl (at least not without permission of all copyright holders/contributers), while `GPL-3.0-or-later` allows anyone to release the software under future versions of the GPL.

**Reply by akurdunkar** [reply]
I think it's a `GPL-3.0 only`.  Also, I have released the first version you can use that.

**Reply by ghost** [reply]
Thank you, unfortunately, I have encountered two issues when packaging:
1) There characters surrounding the queue appear to be missing, is there any particular font that is needed to render these? (after further testing, it seems that it is rendered in bash, just not ash (alpine's default shell).
![image](https://user-images.githubusercontent.com/109069892/218827134-48742c02-1422-4402-b900-2b77bfa6a8e0.png)
2) Whenever gomp is launched, gomp increases the cpu usage of one of my cores to 100%, which I'm guessing is not what happens to most people (It might be to do with the number of songs I have in my library (> 1200)).
Other information:
```
mpd-version: 0.23.12
gomp-version: 1.0.0 (2a2579c)
terminal-emulator: alactitty 0.11.0
shell: ash (busybox 1.36.0) | bash (5.2.15)
```

**Reply by akurdunkar** [reply]
I will take a look into this.

**Reply by ghost** [reply]
Well that's about it then. Getting images to work is out of the scope of the package, since that depends on the terminal itself. Thanks for all the help!

**Reply by akurdunkar** [reply]
Thanks for reporting the issues I have opened two issues. If you get any insights please report there. 🤝

**Reply by ghost** [reply]
I am sorry about this, but I forgot one last thing: what architectures will actually work as expected with gomp?

**Reply by ghost** [reply]
I will just use all arch since there shouldn't be any real problems.

**Reply by akurdunkar** [reply]
Yes. Makes Sense.
